packages/checkout/src/checkout.test.ts

```diff
@@ -474,4 +474,29 @@ describe("@fung-sdk/checkout", () => {
     expect(iframe).not.toBeNull();
     expect(iframe?.src).toContain("wallets");
   });
+
+  it("should return correct query parameters", () => {
+    const checkout = new Checkout({
+      checkoutId: "abc",
+      containerId: "xyz",
+      formOnly: true,
+      language: "fr",
+    });
+
+    // @ts-ignore: We are testing a private method
+    const queryParams = checkout.getQueryParameters();
```

Comment:
Why do we need to test private methods? 
Generally it is discouraged as its encapsulated and implementation is unknown.

Reply:
Good catch, will render the iframe and get the url from there to compare